test/integration/home_page_test.rb

```diff
@@ -25,16 +25,10 @@ class HomePageTest < ActionDispatch::IntegrationTest
     assert page.has_content?('You must login')
     click_link 'Lessons'
     assert page.has_no_content?('You must login')
+    # TODO: Implement clicking on track buttons
 
-    Track.all.each do |track|
-      # TODO: Implement clicking on track buttons
-      track.sections.sort_by(&:name).each do |section|
-        section.lessons.sort_by(&:position).each do |lesson|
-          click_link lesson.name[0..-2]
-          assert page.has_content?('You must login')
-        end
-      end
-    end
+    click_link Track.all.first.sections.first.lessons.first.name[0..-2]
```

Comment:
I prefer this, it's more readable, even it's hard coded
```suggestion
    click_on 'React'
    click_on 'Design system'
    assert_text 'You must login'
```